src/Core/Layer/Layer.js

```diff
@@ -1,5 +1,77 @@
 import { EventDispatcher } from 'three';
 
+/**
+ * Layer's sequence (index) event
```

Comment:
`sequence` is not really clear from an outside POV. We are talking about the order in the stack right? Maybe add a little phrase explaining that.